### DISCUSSION TITLE
Safetensor not working ?

### DISCUSSION THREAD

**Discussion by hackeranmol** [chidiwilliams/buzz#968] [Q&A] [answered]
I want to run this custom model - https://huggingface.co/Garon16/whisper_small_ru_f for a better russian transcription 

But its in safetensor format ? Does buzz only support .pt format ?

**Comment by raivisdejus** [comment, marked as answer]
Both formats are supported in Huggingface whisper type.

Buzz will check if the safetensor model file is available and if so will use it, if it is not available it will try to load other supported model types. So any Huggingface whisper repo should work.

If some does not work, please let us know.
